themes/admin/Sea_Green/css/sea_green.css

```
@@ -17765,6 +17547,26 @@ a {
 
+a {
+  color: #14AE5C;
+  font-weight: 500;
+}
+a.ls-link:hover {
+  color: #14AE5C;
+  text-decoration: underline;
+}
+a.ls-link:focus-visible {
+  padding: 1px 4px;
+  outline: none;
```

Stylelint (17.11.0)

[error] Duplicate property "outline" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)